parking_permits_app/models/refund.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.gis.db import models
+from django.utils.translation import gettext_lazy as _
+
+from .. import constants
+from . import Customer, ParkingPermit
+from .mixins import TimestampedModelMixin, UUIDPrimaryKeyMixin
+
+
+class Refund(TimestampedModelMixin, UUIDPrimaryKeyMixin):
+    customer = models.ForeignKey(
+        Customer,
+        verbose_name=_("Customer"),
+        on_delete=models.PROTECT,
+        blank=False,
```

Comment:
IMO it's better to leave out `blank=False` and `null=False`, less code is always better if we can achieve the same things :). Also, these are meaningful defaults, and we should keep the field definition consistent, i.e. leave them out for all fields that not null and not blank fields.

Reply:
Good point. Fixed